docker/README.md

```diff
@@ -0,0 +1,31 @@
+# Run Unit tests
+
+Requires Python packages installed by `Dockerfile`. Look for `
```

Comment:
Maybe a Makefile could include all these steps? the virtualenv could be created on a folder inside the project so everything remains self-contained

Reply:
Great idea.